systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -363,6 +373,50 @@ public void testForUpdateValuesInConfigMap() {
         }
     }
 
+    @Test
+    @Resources(value = "../examples/templates/cluster-controller", asAdmin = true)
+    @OpenShiftOnly
+    public void testSendMessages() {
+        String topicName = "test-topic";
+        Oc oc = (Oc) this.kubeClient;
+        int messagesCount = 5;
+        oc.namespace();
+        String clusterName = "openshift-my-cluster";
+        oc.newApp("strimzi-ephemeral", map("CLUSTER_NAME", clusterName));
+        oc.waitForStatefulSet(zookeeperStatefulSetName(clusterName), 1);
+        oc.waitForStatefulSet(kafkaStatefulSetName(clusterName), 3);
+        String bootstrapServers = "";
+        for (int i = 0; i < 3; i++) {
+            String kafkaPodJson = oc.getResourceAsJson("pod", kafkaPodName(clusterName, i));
+            bootstrapServers = bootstrapServers + JsonPath.parse(kafkaPodJson).read("$.status.podIP").toString() + ":9092,";
+        }
+        bootstrapServers = bootstrapServers.substring(0, bootstrapServers.length() - 1);
+
+        String yaml = getContent(new File("../examples/configmaps/topic-controller/kafka-topic-configmap.yaml"), node -> {
+            JsonNode metadata = node.get("metadata");
+            ((ObjectNode) metadata).put("name", topicName);
+            JsonNode labels = metadata.get("labels");
+            ((ObjectNode) labels).put("strimzi.io/cluster", clusterName);
+            JsonNode data = node.get("data");
+            ((ObjectNode) data).put("name", topicName);
+        });
+        oc.createContent(yaml);
+        oc.waitForDeployment(topicControllerName(clusterName));
+
+        KafkaProducer producer = new KafkaProducer("my-topic", bootstrapServers);
```

Comment:
Won't the change to using the broker service DNS name as the advertised listener mean that this test is about to break because it will no longer be possible to reach the pod from outside the cluster?